CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+0.27.1 / TBA
```

Comment:
I'd say the change is more at the level of a feature than a bug fix. Can this be updated to `0.28.0` please?